Overlay/style.css

```diff
@@ -0,0 +1,12 @@
+@import '../variables.css';
+
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: var(--outer-space-ultra-dark);
+  overflow-x: hidden;
+  z-index: 25;
```

Comment:
I think we need to keep these z-indexes in a file with variables 😄, that way we can see all z-index's in one place

I've seen the concept of `layers` work pretty well here

```
0000s: (optional)navbar
1000s: overlays
2000s: modals
3000s: tooltips
```